weaviate/graphql/nearvectorbuilder_test.go

```diff
@@ -78,7 +79,7 @@ func TestNearMultiVectorBuilder_build(t *testing.T) {
 
 	t.Run("No combination with vector per target", func(t *testing.T) {
 		vector := NearVectorArgumentBuilder{}
-		str := vector.WithVectorPerTarget(map[string][]float32{"one": {1, 2, 3}, "two": {4, 5, 6}}).build()
+		str := vector.WithVectorPerTarget(map[string]models.Vector{"one": []float32{1, 2, 3}, "two": []float32{4, 5, 6}}).build()
```

Comment:
Does this change constitute a breaking change? If so, is it one that we have to tolerate due to the tight server/client coupling?

Reply:
yes, it's a breaking change, but a really simple one to resolve